modules/ROOT/pages/workspaces-and-repositories.adoc

```diff
@@ -298,3 +298,10 @@ To restore an older point:
 . Go to the *Team* > *SVN*, then *Manage restore points*.
 . Select *Restore to an older point* then click *_Next_*.
 . Select the restore point you want to restore, then click *_Restore_*. The project reverts to the restore point.
+
+== image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
+
+NOTE: Since 2021.2-u1
```

Comment:
```suggestion
```
Useless, you add this on the 2021.1 branch, older versions won't have the section.

Reply:
It won't be available in 2021.2-u0, so I need to explicitly specify the maintenance version, no ?